Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -489,6 +500,10 @@ void ProgramShaderCache::Shutdown()
 			u8 *binary = data + sizeof(GLenum);
 			GLenum *prog_format = (GLenum*)data;
 			glGetProgramBinary(entry.second.shader.glprogid, binary_size, nullptr, prog_format, binary);
+			if (glGetError() != GL_NO_ERROR)
+			{
+				continue;
```

Comment:
How would you use a vector here? Force one to allocate enough data and then give a pointer to the first element?